### PR TITLE
fix(DB/Creature): Add missing spell to Infected Kodo Beast & disable gravity for Column Ornament

### PR DESCRIPTION
**Pull request by avarishd**
… Ornament

## Changes Proposed:
-  Add Stampede ability to Infected Kodo Beast(s).
-  Disable gravity for Ornament(s) so they don't fall to the ground.

## Tests Performed:

- tested in-game

## How to Test the Changes:

Infected Kodo Beast 
.q a 11690
.go c id 25596
Notice the spells

Column Ornament
.go c id 29754
Check if they are staying mid-air and not falling to the ground.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Maelthyrr** [comment]
Hi, tested and look fine. Thanks for the PR.

File: data/sql/updates/pending_db_world/rev_1676663461812301700.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Infected Kodo Beast - add missing ability, Stampede.
+DELETE FROM `creature_template_spell` WHERE `CreatureID`=25596;
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
+(25596, 0, 45876, 12340),
+(25596, 1, 45877, 12340);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1676663461812301700.sql:5]: Creature_template_spell only serves for charmed creature spells. You know, the spells they can use when humanoids are charmed or demons are enslaved.

To make them cast it they need to be added to the SAI script

File: data/sql/updates/pending_db_world/rev_1676663461812301700.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+-- Column Ornament - disable gravity, previously falling to the ground.
+UPDATE `creature_template_movement` SET `Flight` = 1 WHERE `Flight` = 0 AND `CreatureId` = 29754;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1676663461812301700.sql:8]: this part isn't needed 😛 
```suggestion
UPDATE `creature_template_movement` SET `Flight` = 1 WHERE `CreatureId` = 29754;
```

File: data/sql/updates/pending_db_world/rev_1676663461812301700.sql (hunk 1 above)
Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1676663461812301700.sql:5]: And for vehicles creatures, which is this case.

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1676663461812301700.sql:5]: Oop, sorry lol

**Comment by Gultask** [comment]
Verified Build correlates with TC's DB

**Review by Gultask — APPROVED** [review]

**Pull request merged**